src/components/PopupWithImage.js

```diff
@@ -6,10 +6,10 @@ export default class PopupWithImage extends Popup {
     this._imagePreview = document.querySelector(".modal__preview-image");
     this._modalImageText = document.querySelector(".modal__preview-title");
   }
-  openModal({ name, link }) {
-    this._imagePreview.src = link;
-    this._imagePreview.alt = `Photo of ${name}`;
-    this._modalImageText.textContent = name;
+  openModal(card) {
+    this._imagePreview.src = card.link;
+    this._imagePreview.alt = `Photo of ${card.name}`;
+    this._modalImageText.textContent = card.name;
     super.openModal();
   }
   closeModal() {
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.